twitter.html.original.html

```diff
@@ -0,0 +1,175 @@
+        <div class="post__avatar">
+          <img
+            src=https://image.freepik.com/free-vector/man-profile-cartoon_18591-58482.jpg
+          />
+        </div>
+
+        <div class="post__body">
+          <div class="post__header">
+            <div class="post__headerText">
+              <h3>
+                Khateeb 
+                <span class="post__headerSpecial"
+                  ><span class="material-icons post__badge"> verified </span>@Khateeb</span
+                >
+              </h3>
+            </div>
+            <div class="post__headerDescription">
+              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <img
+            src="https://www.focus2move.com/wp-content/uploads/2020/01/Tesla-Roadster-2020-1024-03.jpg"
+            alt=""
+          />
+          <div class="post__footer">
+            <span class="material-icons"> repeat </span>
+            <span class="material-icons"> favorite_border </span>
+            <span class="material-icons"> publish </span>
+          </div>
+        </div>
+      </div>
+      
+
+      <div class="post">
+        <div class="post__avatar">
+          <img
+            src=https://image.freepik.com/free-vector/man-profile-cartoon_18591-58482.jpg
+            alt=""
+          />
+        </div>
+
+        <div class="post__body">
+          <div class="post__header">
+            <div class="post__headerText">
+              <h3>
+              Khateeb
+                <span class="post__headerSpecial"
+                  ><span class="material-icons post__badge"> verified </span>@Khateeb</span
+                >
+              </h3>
+            </div>
+            <div class="post__headerDescription">
+              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <img
+            src="https://www.focus2move.com/wp-content/uploads/2020/01/Tesla-Roadster-2020-1024-03.jpg"
+            alt=""
+          />
+          <div class="post__footer">
+            <span class="material-icons"> repeat </span>
+            <span class="material-icons"> favorite_border </span>
+            <span class="material-icons"> publish </span>
+          </div>
+        </div>
+      </div>
+      
+    </div>
+  
+
+    <!-- Widgets -->
+    <div class="widgets">
+      <div class="widgets__input">
+        <span class="material-icons widgets__searchIcon"> search </span>
+        <input type="text" placeholder="Search Twitter" />
+      </div>
+
+      <div class="widgets__widgetContainer">
+        <h2>What's happening?</h2>
+        <blockquote class="twitter-tweet">
+          &mdash; Cristiano Ronaldo
+          <a href="https://twitter.com/Cristiano/status/1461729514749366281"
+            >Nov 19, 2021</a
+          >
+        </blockquote>
+        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
It looks like you're using a widget from Twitter here. This is a good thing to do if you were trying to show a tweet inside of a website you owned; however, we were looking for you to create the tweet yourself using your own HTML/CSS